bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/core/listeners/SetupListener.java

```diff
@@ -56,7 +56,7 @@ private boolean compatible(Alloc pred, Alloc succ) {
 		return 
 			pred != null &&
 			pred.getObs().getInstrumentString().equals(succ.getObs().getInstrumentString()) &&
-			pred.getObs().getCoords().equals(succ.getObs().getCoords());
+			pred.getObs().getCoords(pred.getMiddlePoint()).equals(succ.getMiddlePoint());
```

Comment:
For a non-sidereal target in theory this will never be equals, but maybe that just means you always need setup for two non-sidereal observations even if they are the "same" target?  I don't know the answer to this one.